lib/apexcharts/options/legend.rb

```diff
@@ -1,24 +1,25 @@
 module ApexCharts
   class LegendOptions < ::SmartKv
     optional *%i[
-      containerMargin
+      show
+      showForSingleSeries
+      showForNullSeries
+      showForZeroSeries
+      position
+      horizontalAlign
       floating
-      fontFamily
       fontSize
+      fontFamily
+      width
+      height
       formatter
-      horizontalAlign
-      itemMargin
-      labels
-      markers
       offsetX
       offsetY
+      labels
+      markers
+      itemMargin
       onItemClick
       onItemHover
-      position
-      show
-      showForSingleSeries
-      textAnchor
```

Comment:
`textAnchor` is also valid: https://github.com/apexcharts/apexcharts.js/blob/512148c1462a4c8c9d3beca4988a6cc36af4b99b/types/apexcharts.d.ts#L559